### PR TITLE
[EfficientNetV2 Backbone] Style fix

### PR DESCRIPTION
**Pull request by cosmo3769**
# What does this PR do?

Matches the same style as other backbones.

## Who can review?

@jbischof @ianstenbit

### CODE REVIEW

File: keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_aliases.py
Comment (jbischof) [inline keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_aliases.py:1]: We should also remove the no-weight presets on the "B" family of models

**Review by jbischof — COMMENTED** [review]
Thank you! One nit

File: keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_aliases.py
Comment (cosmo3769) [reply keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_aliases.py:1]: Umm, B0, B1, B2 models have preset_with_weights. B3 model don't have so I removed the no-weight preset only on this. Do I have to remove in all? 🤔

Comment (jbischof) [reply keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_aliases.py:1]: Here's an example of the needed change:

OLD
```python
    @classproperty
    def presets(cls):
        """Dictionary of preset names and configurations."""
        return {
            "efficientnetv2_b2": copy.deepcopy(
                backbone_presets["efficientnetv2_b2"]
            ),
        }
    @classproperty
    def presets_with_weights(cls):
        """Dictionary of preset names and configurations that include
        weights."""
        return {
            "efficientnetv2_b2_imagenet": copy.deepcopy(
                backbone_presets_with_weights["efficientnetv2_b2_imagenet"]
            ),
        }
```

NEW
```python
    @classproperty
    def presets(cls):
        """Dictionary of preset names and configurations."""
        return {
            "efficientnetv2_b2_imagenet": copy.deepcopy(
                backbone_presets_with_weights["efficientnetv2_b2_imagenet"]
            ),
        }
    @classproperty
    def presets_with_weights(cls):
        """Dictionary of preset names and configurations that include
        weights."""
        return cls.presets
``` 

Or check out one of our existing backbones ([example](https://github.com/keras-team/keras-cv/blob/1eabccb30908a1a10143c0c46204443d7c7825d5/keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_aliases.py#L104)).

Comment (cosmo3769) [reply keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_aliases.py:1]: Ooh understood. Thank you.

**Review by jbischof — APPROVED** [review]

**Pull request merged**